cogs/admin.py

```diff
@@ -0,0 +1,40 @@
+from asgiref.sync import sync_to_async
+from discord.ext import commands
+from discord_slash import cog_ext
+from discord_slash.utils.manage_commands import create_option
+from core.models.guild import Guild
+from discord.ext.commands.errors import MissingRole
+
+class admin(commands.Cog):
+
+    def __init__(self, bot):
+        self.bot = bot
+
+
+    @cog_ext.cog_subcommand(base="admin", name="set_withdrawl_address", description="Set withdrawl adress for the server", options=[
+        create_option(name="address", description="Withdrawl address", required=True, option_type=3)
+    ])
+    async def admin_set_withdrawl_address(self, ctx, address):
+
+        has_role = False
+
+        for role in ctx.author.roles:
+            if role.name == "Maakay Bot Admin":
```

Comment:
Let's not use the role name to verify if the user has correct permissions since the server owner can change the name of the role to whatever they wish for.

We already have a field in the Guild table `manager_role_id` from where we can check the permision.